app/admin/create/page.tsx

```diff
@@ -79,7 +120,7 @@ export default function Create() {
   const handleAddQuestion = () => {
     setFormData((prevData) => ({
       ...prevData,
-      questions: [...prevData.questions, { question: "", additional: "" }],
+      questions: [...prevData.questions, { question: "", context: "" }],
```

Comment:
Does changing the key of the object affect other listing features (like viewing a feature)? I imagine it wouldn't since we actually don't use context on other pages 😅

Reply:
This change was part of a larger fix to the type of questions. It was defined differently in different locations, so I had to go through the places where questions were used and make them consistent. They all look like `{question: "", context: ""}` now, so this shouldn't be a problem. Also, since I modified Form.tsx, we should see the addition of the grayed-out context inside the text box in every instance where a Form is used.